src/pkg/podman/podman.go

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2019 – 2022 Red Hat Inc.
+ * Copyright © 2023 Red Hat Inc.
```

Comment:
In one of the previous comments you told me it should be 2023 instead of 2019-2022 https://github.com/containers/toolbox/pull/1208#discussion_r1087708243
Should I leave it then just the way it was?

Reply:
Yes, that was for `src/pkg/skopeo/skopeo.go`, but this is `src/pkg/podman/podman.go`.

Usually, you only update the copyright notices in the files that you touch.

Reply:
My fault, I didn't read the file correctly, thank you!